prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-presubmits.yaml

```diff
@@ -128,8 +128,8 @@ presubmits:
           readOnly: true
         resources:
           requests:
-            # When running at parallel=15, we use 20Gi. So request 40Gi to be safe.
-            memory: "40Gi"
+            # When running at parallel=15, we use 20Gi. So request 80Gi to be safe.
```

Comment:
"we use 20 so request 80" sounds funny. Do you know how much it uses now?

Reply:
I haven't measured how much it uses now, so I can't say for sure. At any rate our nodes are underutilized so it should be a safe change